### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by prestosole**

### CODE REVIEW

**Comment by triallax** [comment]
Please rebase and check the latest versions again, in case there are new versions now.

**Comment by prestosole** [comment]
Done! No new versions

**Comment by triallax** [comment]
I know for a fact that that is not correct, as lint has 1.7.2 now.

**Comment by triallax** [comment]
If you have some time, can you please fix the points I mentioned in my previous comments? This PR will fix the failing builds we have right now.

**Comment by prestosole** [comment]
Resolved!

**Comment by triallax** [comment]
You don't have to fix the deprecation warnings; these can be fixed in a separate PR.

File: packages/clima_ui/analysis_options.yaml [hunk 1]
```diff
@@ -38,3 +38,6 @@ linter:
 
     # Very infuriating and unnecessary
     sort_pub_dependencies: false
+
+    # Still in beta as of v1.6.0
```
Comment (triallax) [inline packages/clima_ui/analysis_options.yaml]: Version v1.6.0 of what?

Comment (prestosole) [reply packages/clima_ui/analysis_options.yaml]: Of Lint, but you reminded me. I forgot to check if it's out of beta now in v1.7.2. Can you check it out for me please?

Comment (triallax) [reply packages/clima_ui/analysis_options.yaml]: You're misunderstanding; the rule in general is beta, nothing to do with the `lint` package. Please fix the misunderstanding in the comment:
```suggestion
    # Still in beta as of Flutter 2.5
```

**Review by triallax — APPROVED** [review]

**Pull request merged**